applications/GeoMechanicsApplication/custom_elements/U_Pw_small_strain_element.cpp

```diff
@@ -1568,6 +1571,22 @@ Vector UPwSmallStrainElement<TDim, TNumNodes>::CalculateStrain(const Matrix& rDe
                            : this->CalculateCauchyStrain(rB, rDisplacements);
 }
 
+template <unsigned int TDim, unsigned int TNumNodes>
+std::vector<Vector> UPwSmallStrainElement<TDim, TNumNodes>::CalculateStrains(const std::vector<Matrix>& rDeformationGradients,
```

Comment:
I wondered if only the s of strains would be enough to indicate that this is for all integration points ( e.g. for all components. ) CalculateStrain already is for all components in 1 integration point.

I don't know a better name, they soon become very long.

Reply:
I'm also not too sure about better naming unfortunately

Reply:
On second thought, would CalculateStrainVectors be a more suitable name?